Formula/joern.rb

```diff
@@ -0,0 +1,83 @@
+class Joern < Formula
+  desc "Open-source code analysis platform based on code property graphs"
+  homepage "https://joern.io/"
+  url "https://github.com/joernio/joern/archive/refs/tags/v1.1.1384.tar.gz"
+  sha256 "c71b77c519b051a07728e20eb78538f6c76287e7ec2ee4ee082a8fd852f67dad"
+  license "Apache-2.0"
+
+  depends_on "node@16" => :build
```

Comment:
what is the issue of using latest node or node@18?

Reply:
Great catch! When the build originally failed with `depends_on "node"` (currently node 19), I set the formula to use the alternate version available on my machine (node 16) and I forgot to revisit the issue. Here is the build error that results with node 19:

```bash
==> yarn install
yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
[4/5] Linking dependencies...
[5/5] Building fresh packages...
$ pkg . --no-bytecode --no-native-build --public --compress GZip
> pkg@5.8.0
compression:  GZip
> Targets not specified. Assuming:
  node19-linux-arm64, node19-macos-arm64, node19-win-arm64
> Error! No available node version satisfies 'node19'
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
/opt/homebrew/Library/Homebrew/shims/shared/git --version
/opt/homebrew/Library/Homebrew/shims/shared/curl --version
/opt/homebrew/Library/Homebrew/ignorable.rb:29:in `block in raise'
BuildError: Failed executing: yarn install
```

However, there is no issue with node 18, so I will update the formula accordingly.

Reply:
@chenrui333 fixed in 7b106265c93d0c85c26c7a21e8fef716c232d892

Reply:
👍